### ISSUE TITLE
No names at Fields

### ISSUE THREAD

**Issue by Antidote2151** [goauthentik/authentik#11237]
**Describe the bug**
Fields in the default-souce-enrollment-prompt are not displayed correctly, and it cannot be updated.

**To Reproduce**

1. Create a Flow
2. Go into Flow
3. Click on "Stage Bindings"
4. Click on "Bind existing stage"
5. Select "default-souce-enrollment-prompt"
6. Click on "Create"
7. Click on "Edist Stage"
8. Then you see

**Expected behavior**
The names of the fields should be there

**Screenshots**
![image](https://github.com/user-attachments/assets/dd8d6b40-e9c1-4956-8cf5-3c1146e6000c)

**Logs**
```
authentik-server  | {"auth_via": "session", "domain_url": "access.o", "event": "/api/v3/stages/prompt/stages/1854ffef-b4eb-4de6-830e-fe585dea2658/", "host": "access.o", "level": "info", "logger": "authentik.asgi", "method": "GET", "pid": 93, "remote": "217", "request_id": "ace135352bb74082b778d7413c8d41a7", "runtime": 53, "schema_name": "public", "scheme": "https", "status": 200, "timestamp": "2024-09-05T20:51:32.418866", "user": "sup", "user_agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/128.0.0.0 Safari/537.36"}
authentik-server  | {"event":"/socket.io/?EIO=4&transport=polling&t=P73zEF6","host":"stat","level":"info","logger":"authentik.outpost.proxyv2.application","method":"GET","name":"Provider for Status","remote":"193","runtime":"0.466","scheme":"http","size":135,"status":302,"timestamp":"2024-09-05T20:51:33Z","user_agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/127.0.0.0 Safari/537.36"}
authentik-server  | {"auth_via": "session", "domain_url": "access.o", "event": "/api/v3/stages/prompt/stages/1854ffef-b4eb-4de6-830e-fe585dea2658/", "host": "access.o", "level": "info", "logger": "authentik.asgi", "method": "GET", "pid": 93, "remote": "217", "request_id": "dc54227d42ec47deb1437b7bdf1626d3", "runtime": 75, "schema_name": "public", "scheme": "https", "status": 200, "timestamp": "2024-09-05T20:51:33.999099", "user": "sup", "user_agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/128.0.0.0 Safari/537.36"}
authentik-server  | {"auth_via": "session", "domain_url": "access.o", "event": "/api/v3/policies/all/?ordering=name&page=1&page_size=20&search=", "host": "access.o", "level": "info", "logger": "authentik.asgi", "method": "GET", "pid": 94, "remote": "217", "request_id": "0f9a6204c40445a0b2d4e891cffb07a2", "runtime": 87, "schema_name": "public", "scheme": "https", "status": 200, "timestamp": "2024-09-05T20:51:34.011739", "user": "sup", "user_agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/128.0.0.0 Safari/537.36"}
authentik-server  | {"auth_via": "session", "domain_url": "access.o", "event": "/api/v3/stages/prompt/prompts/?ordering=field_name&page=1&page_size=20&search=", "host": "access.o", "level": "info", "logger": "authentik.asgi", "method": "GET", "pid": 93, "remote": "217", "request_id": "dfea5cb3958745d5bc4f038921bcdf17", "runtime": 109, "schema_name": "public", "scheme": "https", "status": 200, "timestamp": "2024-09-05T20:51:34.034965", "user": "sup", "user_agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/128.0.0.0 Safari/537.36"}

```

**Version and Deployment (please complete the following information):**

-   authentik version: 2024.8.0
-   Deployment: docker-compose

**Comment by kevinkellner** [comment]
Same issue here, also using 2024.8.0 with docker-compose deployment. 

Additionally it is not possible to create a prompt stage via the wizard either. 

![grafik](https://github.com/user-attachments/assets/3d17fa6d-75f5-4b16-aedb-66a9f8c30686)

**Comment by BeryJu** [comment]
This is fixed by 2024.8.1 which was released earlier today.
